Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -393,11 +393,6 @@ - (BOOL)distribute:(MSACDistribute *)distribute releaseAvailableWithDetails:(MSA
 
 - (void)distributeNoReleaseAvailable:(MSACDistribute *)distribute {
   NSLog(@"distributeNoReleaseAvailable invoked");
-  UIAlertController *alert = [UIAlertController alertControllerWithTitle:nil
```

Comment:
Why are they removed? How these dialogs are displayed now?

Reply:
These dialogs have been completely removed due to their unnecessariness. Also they were uninformative and conflicting with "Crash reports" alerts.
Now they appearing only in logs.
@DmitriyKirakosyan approved this approach.

Reply:
This dialog is used to check that NoReleaseAvailable callback works, because Demo apps are tested without checking logs.

Reply:
Also, what about the update dialog? Does it affect the sending of logs?